src/main/java/frc/robot/commands/ClimbDistance.java

```diff
@@ -24,6 +24,7 @@
    * Creates a new ClimbDistance.
    */
   private Climb climb;
+  double climbdashboard = SmartDashboard.getNumber("Climb Speed", 0);
```

Comment:
You need to get the distance passed in through the constructor.

Reply:
I am unsure of what the constructor is could you explain that.